integration_test/EmitVerilog/instance-choice.mlir

```diff
@@ -0,0 +1,54 @@
+// REQUIRES: verilator
+// RUN: circt-opt %s -export-split-verilog='dir-name=%t.dir' -o /dev/null
+// RUN: verilator %driver --cc --sv --exe --build -I%t.dir -F %t.dir/filelist.f -o %t.exe --top-module top
+// RUN: %t.exe --cycles 10 2>&1 | FileCheck %s --check-prefix=DEFAULT
```

Comment:
Does this need a full-blow test or is `verilator --lint-only` sufficient? It seems like no. Given that the alternative emission isn't yet implemented, the check can be as simple as putting an `assert(false)` in the module you want to make sure doesn't run.

Nit: You may be able to avoid file output by piping non-split output directly to Verilator. It will take `circt-opt ... | verilator /dev/stdin -top-module top`. The nit is only for consideration.

Reply:
Does this string work on Windows given the use of a forward slash? If not, is the test properly guarded so it doesn't run on the Windows runner? You can run the windows integration test manually from the `Actions` tab at the top.

Reply:
I do want an execution check here, the cost of it should be irrelevant. Replaced slashes with `%{fs-sep}`, that should work.

Reply:
Also, I do want this to become an integration test for multi-file emission, with verilator being told to ingest the correct file lists.